cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2203,6 +2206,53 @@ if(ARROW_WITH_RAPIDJSON)
   endif()
 endif()
 
+
+macro(build_qpl)
+  message(STATUS "Building QPL from source")
+  set(QPL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/qpl_ep/src/qpl_ep-install")
+  if(MSVC)
+      set(QPL_STATIC_LIB_NAME qplstatic.lib)
+  else()
+    set(QPL_STATIC_LIB_NAME libqpl.a)
+  endif()
+  set(QPL_STATIC_LIB "${QPL_PREFIX}/lib64/${QPL_STATIC_LIB_NAME}")
+  set(QPL_CMAKE_ARGS
+      ${EP_COMMON_CMAKE_ARGS}
+      -DCMAKE_BUILD_TYPE=Release
```

Comment:
We don't need this because `EP_COMMON_CMAKE_ARGS` includes `-DCMAKE_BUILD_TYPE`.